about/our-team.php

```diff
@@ -63,48 +54,60 @@
             <!-- co-president -->
             <div class="cell-lg-12 bio-area">
                 <div class="range range-sm-middle">
-                    <div class="cell-md-3"><img src="/images/bio-pics/lauren-caves.jpg" alt="" width="360" height="404" class="img-responsive" />
+                    <div class="cell-md-3"><img src="/images/bio-pics/andrew-hooker.jpg" alt="" width="360" height="404" class="img-responsive" />
                     </div>
                     <div class="cell-md-6">
                         <div class="inset-xl-right-70 inset-xl-left-70 inset-left-15 inset-right-15">
-                            <h6 class="h6-with-small"><p> Lauren Caves</p><span class="small text-silver-chalice">Co-President</span></h6>
-                            <p>She’s a woman of action, eternal optimist, and a passionate speaker. As an undergraduate mechanical engineering student, her enthusiasm for the art of engineering is vast and the hunger for discovery drives her motivation which has led to many successes during her undergraduate years. Although not easily. She has gone through many failures and complicated situations which shaped a new passion to inspire the youth and other students to overcome diversity and really fight for their dreams. She believes everyone is destined for greatness, and that it's important to not let the minor roadblocks we stumble upon crush the stars we wish upon. </p>
+                            <h6 class="h6-with-small"><p> Andrew Hooker</p><span class="small text-silver-chalice">Co-President</span></h6>
+                            <p></p>
```

Comment:
We don't have a bio written for Andy, and I don't think we'll have one soon so you can go ahead and remove his section and we'll add it later if needed.